Sources/TableViewDriver.swift

```diff
@@ -72,6 +72,8 @@ open class TableViewDriver: NSObject {
 
     private let _automaticDiffingEnabled: Bool
 
+    private let _registerViews: (TableViewModel) -> Void
```

Comment:
This is needed to avoid `associatedtype` issues with `CellContainerViewProtocol`.